requirements.txt

```diff
@@ -1,2 +1,3 @@
 dwave-ocean-sdk>=4.1
-pandas~=2.0
+pandas==1.3.5; python_version < '3.8'
```

Comment:
Just chiming in to say that codegrade is still using 3.7 :roll_eyes: so we're a bit stuck with this for now.